### PR TITLE
New element in section 2 on gathering of data and preparation of datasets for training and evaluation

### PR DESCRIPTION
**Pull request by bowni**
Addresses issue #149

### CODE REVIEW

**Comment by bowni** [comment]
Arf, this should have been a draft PR for now - still have to do the English version

**Comment by bowni** [comment]
Hey @ClementMayer @meuce @celinejacques @SaboniAmine @RomainGoussault could one of you check and validate this PR to finalize the 2021H2 release candidate?
We'll circulate it to the whole communities later today or tomorrow at a maximum!
Thanks!

**Review by SaboniAmine — APPROVED** [review]

**Review by ClementMayer — APPROVED** [review]

File: assessment_framework_eng.md [hunk 1]
```diff
@@ -294,26 +294,56 @@ Among other things, in some cases a specification of the equity sought between p
...
+- [ ] 2.1.a We operate informally on this subject and relies on the practices of each collaborator involved
+- [ ] 2.1.c Our approach includes methods to prevent poisoninck attacks when collecting and gathering data samples
+- [ ] 2.1.b Our approach includes methods to check and make sure when necessary that datasets include samples of rare events
+- [ ] 2.1.d Our approcah includes methods to complete missing values in datasets
+- [ ] 2.1.e Our approach includes methods to handle erroneous or atypical data samples values
+
+<details>
+<summary>Expl2.1 :</summary>
+
+Obtaining and preparing the dataasets is a core acitivity in every data science project. Each data point can have an impact on the learning, and it is thus crucial to define and implement a conscious, coherent, concerted approach to mitigate the risk of learning and testing on problematic datasets.
```
Comment (SaboniAmine) [inline assessment_framework_eng.md:313]: Obtaining and preparing _datasets_

File: assessment_framework_eng.md [hunk 2]
```diff
@@ -294,26 +294,56 @@ Among other things, in some cases a specification of the equity sought between p
...
+
+</details>
+
+<details>
+<summary>Resources2.1 :</summary>
+
+- (Technical guide) *[Tour of Data Sampling Methods for Imbalanced Classification](https://machinelearningmastery.com/data-sampling-methods-for-imbalanced-classification/)*
+- (Software & Tools) *[Pandas Profiling](https://github.com/pandas-profiling/pandas-profiling): Create HTML profiling reports from pandas `DataFrame` objects. The pandas `df.describe()` function is great but a little basic for serious exploratory data analysis. `pandas_profiling` extends the pandas `DataFrame` with `df.profile_report()` for quick data analysis*
+
```
Comment (SaboniAmine) [inline assessment_framework_eng.md:322]: alt: for _extensive_ exploratory

File: assessment_framework_eng.md [hunk 3]
```diff
@@ -346,33 +376,33 @@ In certain cases it is obvious if this risk has to be considered or not (e.g. pr
...
-- [ ] 2.3.c We carry out evaluations on test data from different sub-populations in order to identify possible problematic biases
-- [ ] 2.3.d We select and implement one or more justice and equity measure(s) (_fairness metrics_)
-- [ ] 2.3.e We use _data augmentation_ or _re-weighting_ approaches to reduce possible biases in the data sets
-- [ ] 2.3.f The above practices that we implement are duly documented and integrated into the model lifecycle documentation of the models concerned
-- [ ] 2.3.g We have not yet put in place any such measures
+- [ ] 2.4.a We are not involved in cases where AI models are used in thematic environments with risks of discrimination against certain social groups (gender, origin, age, etc.) | _(Concerned / Not concerned)_
+- [ ] 2.4.b We pay particular attention to the identification of protected attributes and their possible proxies (e.g. studying one by one the variables used as model inputs to identify the correlations they might have with sensitive data)
```
Comment (SaboniAmine) [inline assessment_framework_eng.md:389]: alt : We pay _special_ attention

**Review by SaboniAmine — CHANGES_REQUESTED** [review]
Typo & alternative phrasing suggestions

File: assessment_framework_eng.md (hunk 3 above)
Comment (bowni) [reply assessment_framework_eng.md]: Done

File: assessment_framework_eng.md (hunk 2 above)
Comment (bowni) [reply assessment_framework_eng.md:322]: Done

File: assessment_framework_eng.md (hunk 1 above)
Comment (bowni) [reply assessment_framework_eng.md]: Done

**Comment by bowni** [comment]
Thanks @SaboniAmine , changes done, I let you finalize your review and approve!

**Review by ClementMayer — APPROVED** [review]

**Review by SaboniAmine — APPROVED** [review]

**Pull request merged**